src/frontend/src/components/authenticateBox.ts

```diff
@@ -163,7 +163,7 @@ export const authenticateBoxFlow = async <T, I>({
     pinIdentityMaterial: I;
   }) => Promise<"valid" | "expired">;
   registerFlowOpts: Parameters<typeof registerFlow<T>>[0];
-}): Promise<LoginFlowResult<T> & { newAnchor: boolean }> => {
```

Comment:
Before it returned a promise with the parameter `newAnchor`?

Reply:
before it returned a `newAnchor` field always, now only in the success case (see description)